npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/registry_finder.rb

```diff
@@ -279,6 +292,12 @@ def parsed_yarnrc_yml
 
           @parsed_yarnrc_yml = YAML.safe_load(yarnrc_yml_file.content)
         end
+
+        def normalize_configured_registry(url)
+          url.sub(%r{/+$}, "")
+             .sub(%r{^.*?//}, "")
+             .gsub(/\s+/, "%20")
+        end
```

Comment:
If you have a spec testing this then please point me to it

Reply:
I added this to respect the format currently returned by `RegistryFinder#registry`. The format is a URL with no protocol, no trailing slashes, and escaped espaces. However, they may appear in other format in rc files, so needs to be normalized. There are many specs that fail if you don't do this.

Note that I did not add this logic, it was already there, I just extracted it to method so that I can reuse it.